docs/Customs Declare Exports AutoComplete.js

```diff
@@ -424,23 +433,25 @@ function officeOfExit(){
             case 'F':
             case 'K':
             case 'Y':
-                selectFromAutoPredict(document.getElementById('officeId-container'), "GB000041");
+                document.querySelector("#officeId-container input").value = 'GB000041';
                 break;
             case 'J':
-                selectFromAutoPredict(document.getElementById('officeId-container'), "GB000054");
+                document.querySelector("#officeId-container input").value = 'GB000054';
                 break;
             case 'Z':
-                selectFromAutoPredict(document.getElementById('officeId-container'), "GB000051");
+                document.querySelector("#officeId-container input").value = 'GB000051';
                 break;
             default:
-                selectFromAutoPredict(document.getElementById('officeId-container'), "GB000434");
-        }
-
-        if(document.getElementById("circumstancesCode")) {
-            selectRadioOption(document.getElementById("circumstancesCode"), 1);
+                document.querySelector("#officeId-container input").value = 'GB000434';
         }
 
-        document.getElementsByClassName('button')[0].click()
+        setTimeout(() => {
+            document.getElementById('officeId__option--0').dispatchEvent(new Event("click"));
+            if(document.getElementById('Yes')) {
+                document.getElementById('Yes').click()
+            }
```

Comment:
Why only in this case you're checking if element exists?

Reply:
Because previously we were looking for an id of that radio button. So if the id for "YES" or "No exists then we know that the radio is present.
It is just a quick way to identify if a radio exists there.